websites/O/OpenStreetMap/presence.ts

```diff
@@ -0,0 +1,201 @@
+				if (!locationcity) locationcity = res.address.village;
+				if (!locationcity) locationcity = res.address.town;
+
+				locationstate = res.address.state;
+				locationcountry = res.address.country;
+			})
+			.catch();
+	}
+}, 5000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "logo",
+		startTimestamp: browsingTimestamp,
+		details: "Current page:"
+	};
+
+	if (document.location.pathname.includes("/traces/new"))
+		presenceData.state = "GPS tracks upload";
+	else if (document.location.pathname.includes("/traces"))
+		presenceData.state = "Public GPS tracks";
+	else if (document.location.pathname.includes("/diary/new"))
+		presenceData.state = "Write new diary entry";
+	else if (document.location.pathname.includes("/diary"))
+		presenceData.state = "Users' Diaries";
+	else if (document.location.pathname.includes("/help"))
+		presenceData.state = "Getting Help";
+	else if (document.location.pathname.includes("/welcome"))
+		presenceData.state = "OpenStreetMap basics guide";
+	else if (document.location.hostname.startsWith("welcome"))
+		presenceData.state = "Welcome Mat - Organizations";
+	else if (document.location.pathname.includes("/copyright"))
+		presenceData.state = "Copyright and License";
+	else if (document.location.pathname.includes("/about"))
+		presenceData.state = "About OpenStreetMap";
+	else if (document.location.pathname.includes("/dashboard"))
+		presenceData.state = "User Dashboard";
+	else if (document.location.pathname.includes("/preferences"))
+		presenceData.state = "User preferences";
+	else if (document.location.pathname.includes("/export"))
+		presenceData.state = "Map exports";
+	else if (document.location.pathname.includes("/account/edit"))
+		presenceData.state = "User settings";
+	else if (document.location.pathname.includes("/directions"))
+		presenceData.state = "Planning a route";
+	else if (document.location.pathname.includes("/oauth2"))
+		presenceData.state = "User OAuth applications";
+	else if (document.location.pathname.includes("/history"))
+		presenceData.state = "Changeset history";
+	else if (document.location.hostname.startsWith("dmca"))
+		presenceData.state = "Claim of Copyright Infringement";
+	else if (document.location.hostname.startsWith("dns"))
+		presenceData.state = "GeoDNS Zones";
+	else if (document.location.hostname.startsWith("git"))
+		presenceData.state = "Project Git";
+	else if (document.location.hostname.startsWith("lists"))
+		presenceData.state = "Project Mailing Lists";
+	else if (document.location.hostname.startsWith("blogs"))
+		presenceData.state = "Community blogs";
+	else if (document.location.hostname.startsWith("help")) {
+		presenceData.details = "Reading help:";
+		presenceData.state = document.title.split(" -")[0];
+	} else if (document.location.hostname.startsWith("blog")) {
+		presenceData.details = "Reading a blog post:";
+		presenceData.state = document.title.split(" |")[0];
```

Comment:
I know it works the same, but why the space
```suggestion
		presenceData.state = document.title.split("|")[0];
```